src/background/services/openPayments.ts

```diff
@@ -906,7 +908,16 @@ export class OpenPaymentsService {
 
   async reconnectWallet() {
     try {
+      const { walletAddress } = await this.storage.get(['walletAddress']);
+      const tabId = await this.addPublicKeyToWallet(walletAddress!);
```

Comment:
Lets do a proper check for the wallet address instead of performing a non-null assertion.

Reply:
Might be better to update `Storage` type? When wallet is in `connected` state, certain properties in `Storage` will not be nullish. 
Or maybe better to have runtime checks so user messing with storage (or something else going wrong) still gives us better errors?